flask_restplus/representations.py

```diff
@@ -1,18 +1,32 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals, absolute_import
 
-try:
-    from ujson import dumps
-except ImportError:
-    from json import dumps
+import importlib
+
+from json import dumps
 
 from flask import make_response, current_app
 
+serializer = None
+
 
 def output_json(data, code, headers=None):
     '''Makes a Flask response with a JSON encoded body'''
 
+    global serializer
+
     settings = current_app.config.get('RESTPLUS_JSON', {})
+    custom_serializer = current_app.config.get('RESTPLUS_JSON_SERIALIZER', None)
+
+    # If the user wants to use a custom serializer, let it be
+    if serializer is None and custom_serializer:
+        try:
+            serializer = importlib.import_module("{}.dumps".format(custom_serializer) if '.' not in custom_serializer
```

Comment:
This import is failing for me, for example setting `RESTPLUS_JSON_SERIALIZER='ujson'`  causes a `ModuleNotFoundError`. This can be replicated as such:

```python
import importlib

importlib.import_module("ujson.dumps")
# ModuleNotFoundError: No module named 'ujson.dumps'; 'ujson' is not a package
```

This because `ujson.dumps` is a function, not a module. A solution could be:
```python
import importlib

if serializer is None and custom_serializer is not None:
        try:
            # user provided a module path, attempt to import the 'dumps' method of the module
            mod = importlib.import_module(custom_serializer)
            if hasattr(mod, "dumps"):
                serializer = getattr(mod, "dumps")
        except ImportError:   
             if "." in custom_serializer:
                # user may have provided a path to a function
                # attempt to import the module and retrieve the function attribute
                parts = custom_serializer.split(".")
                mod_path = "".join(parts[:-1])
                func_name = parts[-1]
                try:
                    mod = importlib.import_module(mod_path)
                    if hasattr(mod, func_name):
                        serializer = getattr(mod, func_name)
                except ImportError:
                    pass
    elif serializer is None:
        serializer = dumps
```

Is it also worth logging a warning if the import fails so the user knows their custom serializer isn't being imported?

Reply:
Thanks for your careful review @SteadBytes!
I believe I have now fixed the issue and I added a few more tests to make sure the imported *serializer* is the expected one.

I also added a note in the doc stating that we will always silently fallback to `json.dumps` if we cannot manage to import the configured one.

We might add a more explicit warning thrown by the `output_json` func in the future.